src/analysis/retail/priest/holy/modules/talents/TopRow/BurningVehemence.tsx

```diff
@@ -0,0 +1,56 @@
+import { formatThousands } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/priest';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent, DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+const DAMAGE_INCREASE_FROM_BURNING_VEHEMENCE = 0.25;
+
+class BurningVehemence extends Analyzer {
+  holyFireDamage = 0;
+  holyFireCasts = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.BURNING_VEHEMENCE_TALENT.id);
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.HOLY_FIRE), this.onCast);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.HOLY_FIRE), this.onDamage);
+  }
+
+  onCast(event: CastEvent) {
+    this.holyFireCasts += 1;
+  }
+
+  onDamage(event: DamageEvent) {
+    this.holyFireDamage += event.amount || 0;
```

Comment:
```suggestion
    this.holyFireDamage += event.amount || 0 + event.absorbed || 0;
```

include absorbed damage to account for enemies that get shields.